server/src/main/java/io/druid/client/cache/Cache.java

```diff
@@ -39,6 +40,8 @@
 
   public boolean isLocal();
 
+  public Collection<String> getNamespaces();
```

Comment:
hmmm... it seems this class already had notion of a "namespace". should we document that this not necessarily the same namespace in the sense of lookup datasource?